src/utils.ts

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export function isNavbarItem(child: React.ReactNode): boolean {
```

Comment:
In real projects, when a component is built, every file related to it is under the same folder, here's an example:

```
components/
  Navbar
    Navbar.tsx
    Navbar.scss
    Navbar.tests..tsx
    Navbar.utils.ts
    Navbar.*.ts 
```

This way it's much easier to maintain the component and move it no other folders/projects if needed. Gotcha?